pkgs/by-name/va/vacuum-go/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, vacuum-go }:
+
+buildGoModule rec {
+  pname = "vacuum-go";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "daveshanley";
+    repo = "vacuum";
+    # using rev because by version gives: 'the given path has multiple possibilities' error
+    rev = "4a996f877ef6738c0615d0c81b9f19dfeb248ee8";
+    hash = "sha256-YQJKmLhxBnU6gKbhnzVAF53N1qS0/DQjjuOj8g6y+vo=";
+  };
+
+  vendorHash = "sha256-OhdN4/fNbXa5ZMakdf370rqyDlCVYjJ1IfeV6hEwcv4=";
+
+  CGO_ENABLED = 0;
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.version=v${version}"
+  ];
+
+  subPackages = [ "./vacuum.go" ];
+
+  passthru = {
+    tests.version = testers.testVersion {
+      package = vacuum-go;
+      command = "vacuum version";
+      version = "v${version}";
+    };
+  };
+
+  meta = with lib; {
```

Comment:
cold you avoid using `with lib` here because it is a problematic anti-pattern, refer to: https://github.com/NixOS/nixpkgs/issues/208242

Reply:
should be better now